pkg/util/uint160.go

```diff
@@ -123,11 +123,11 @@ func (u *Uint160) UnmarshalJSON(data []byte) (err error) {
 		return err
 	}
 	js = strings.TrimPrefix(js, "0x")
-	*u, err = Uint160DecodeStringBE(js)
+	*u, err = Uint160DecodeStringLE(js)
```

Comment:
@roman-khimov Which one should we use here?

Reply:
LE, that's the whole point of #769.